### PR TITLE
[12.0][BACKPORT] account_move_post_date_user

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
Backport of account_move_post_date_user from 13.0

cc: @JordiMForgeFlow

### CODE REVIEW

File: account_move_post_date_user/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    last_post_date = fields.Datetime(
+        string="Last Posted on", readonly=True, tracking=True
+    )
+    last_post_uid = fields.Many2one(
+        "res.users", string="Last Posted by", readonly=True, tracking=True
+    )
+
+    def post(self, invoice=False):
+        res = super().post()
```
Comment (JordiMForgeFlow) [inline account_move_post_date_user/models/account_move.py]:
```suggestion
        res = super().post(invoice)
```

Comment (LoisRForgeFlow) [reply account_move_post_date_user/models/account_move.py]: invoice=invoice

**Review by JordiMForgeFlow — APPROVED** [review]
code review 👍🏼

**Review by GuillemCForgeFlow — COMMENTED** [review]
LGTM! functional and code review 👍🏿 
A minor comment, perhaps the commit message should be BKP or something that does not relate to a migration?

**Review by LoisRForgeFlow — APPROVED** [review]
ty

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

**Pull request merged**